diff -ur jazz-src-4.1.3/install-jazz jazz-src-4.1.3_patch/install-jazz
--- jazz-src-4.1.3/install-jazz	Sat Mar 11 21:22:52 2000
+++ jazz-src-4.1.3_patch/install-jazz	Fri Aug 30 23:05:10 2002
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 
-DEST=/usr/local/jazz
+DEST=/usr/local/jazz-bin-4.1.3
 
 
 echo -n "installing JAZZ++ to $DEST ... "
diff -ur jazz-src-4.1.3/src/pianowin.cpp jazz-src-4.1.3_patch/src/pianowin.cpp
--- jazz-src-4.1.3/src/pianowin.cpp	Fri Mar  3 20:03:57 2000
+++ jazz-src-4.1.3_patch/src/pianowin.cpp	Fri Aug 30 23:06:08 2002
@@ -1144,7 +1144,7 @@
 
 
 
-void tPianoWin::DrawEvent(tEvent *e, wxBrush *Brush, int xor, int force_color)
+void tPianoWin::DrawEvent(tEvent *e, wxBrush *Brush, int xxor, int force_color)
 {
   // SN++ Aftertouch
   if (e->IsKeyPressure() || e->IsChnPressure()) return;
@@ -1155,11 +1155,11 @@
   if (length < 3)
     length = 3;
   dc->BeginDrawing();
-  if (xor)
+  if (xxor)
     dc->SetLogicalFunction(wxXOR);
   long x = Clock2x(e->Clock);
   long y = Pitch2y(e->GetPitch());
-  if (!xor)	// wxXOR is buggy?!
+  if (!xxor)	// wxXOR is buggy?!
   {
     dc->SetBrush(wxWHITE_BRUSH);
     dc->DrawRectangle(x, y + LittleBit, length, hLine - 2 * LittleBit);
@@ -1179,7 +1179,7 @@
   // end velocity colors
 
   dc->DrawRectangle(x, y + LittleBit, length, hLine - 2 * LittleBit);
-  if (xor)
+  if (xxor)
     dc->SetLogicalFunction(wxCOPY);
   dc->SetBrush(wxBLACK_BRUSH);
   dc->EndDrawing();
diff -ur jazz-src-4.1.3/src/pianowin.h jazz-src-4.1.3_patch/src/pianowin.h
--- jazz-src-4.1.3/src/pianowin.h	Fri Mar  3 20:03:57 2000
+++ jazz-src-4.1.3_patch/src/pianowin.h	Fri Aug 30 23:05:53 2002
@@ -92,7 +92,7 @@
 
     tEventArray PasteBuffer;
     void DrawEvents(tTrack *t, int Stat, wxBrush *Brush, int force_colors);
-    void DrawEvent(tEvent *, wxBrush *Brush, int xor, int force_color=0);
+    void DrawEvent(tEvent *, wxBrush *Brush, int xxor, int force_color=0);
     void DrawPianoRoll();
 
     int OnMouseEvent(wxMouseEvent &e);
diff -ur jazz-src-4.1.3/src/random.h jazz-src-4.1.3_patch/src/random.h
--- jazz-src-4.1.3/src/random.h	Mon Mar 27 20:42:35 2000
+++ jazz-src-4.1.3_patch/src/random.h	Fri Aug 30 23:05:30 2002
@@ -35,7 +35,7 @@
 #endif
 
 // gcc > 2.7.2 does not have ACG anymore?
-#define USE_ACG 1
+//#define USE_ACG 1
 
 #if USE_ACG
 #include <ACG.h>	// random generator from libg++
